### PR TITLE
Make taskEither an instance of BiFunctor

### PR DESCRIPTION
**Pull request by teves-castro**
This PR makes taskEither an instance of BiFunctor.

### CODE REVIEW

File: src/TaskEither.ts [hunk 1]
```diff
@@ -58,6 +59,9 @@ export class TaskEither<L, A> {
   orElse<M>(f: (l: L) => TaskEither<M, A>): TaskEither<M, A> {
     return new TaskEither(this.value.chain(e => e.fold(l => f(l).value, a => eitherTTask.of(a))))
   }
+  bimap<V, B>(f: (l: L) => V, g: (a: A) => B): TaskEither<V, B> {
```
Comment (gcanti) [inline src/TaskEither.ts]: @teves-castro what you think about adding `bimap` to `EitherT`

```ts
export function bimap<F extends URIS2>(
  F: Functor2<F>
): <M, L, V, A, B>(fa: Type2<F, M, Either<L, A>>, f: (l: L) => V, g: (a: A) => B) => Type2<F, M, Either<V, B>>
export function bimap<F extends URIS>(
  F: Functor1<F>
): <L, V, A, B>(fa: Type<F, Either<L, A>>, f: (l: L) => V, g: (a: A) => B) => Type<F, Either<V, B>>
export function bimap<F>(
  F: Functor<F>
): <L, V, A, B>(fa: HKT<F, Either<L, A>>, f: (l: L) => V, g: (a: A) => B) => HKT<F, Either<V, B>>
export function bimap<F>(
  F: Functor<F>
): <L, V, A, B>(fa: HKT<F, Either<L, A>>, f: (l: L) => V, g: (a: A) => B) => HKT<F, Either<V, B>> {
  return (fa, f, g) => F.map(fa, e => e.bimap(f, g))
}
```

and then derive `TaskEither`'s `bimap` method from that, like I did with `mapLeft`?

Comment (teves-castro) [reply src/TaskEither.ts]: That indeed sounds more interesting.

Comment (teves-castro) [reply src/TaskEither.ts]: @gcanti done.

**Comment by gcanti** [comment]
Thanks @teves-castro

**Pull request merged**